src/Blazor.Gitter.Core/Components/Shared/AppState.cs

```diff
@@ -176,7 +189,15 @@ private void SetTimeoutTime()
             }
             TimeoutTime = DateTime.UtcNow.AddMinutes(TIMEOUT);
             TimeoutTimer.Start();
-            TimeoutChanged?.Invoke(this, TimeoutTime);
+            if (!(LastTriggerTimeOutChanged is object))
```

Comment:
This is to prevent too many UI refreshes - as mouseout on a message now fires RecordActivity